packaging/plugins/provision.sh

```diff
@@ -67,17 +58,83 @@ function wagon_create_package(){
         echo "## /packaging/$CONSTRAINTS_FILE doesn't exist"
         wagon create -s ../$PLUGIN_NAME/ -r -v -f
     fi
+
+}
+
+function generate_windows_plugin() {
+    # Print Plugin params
+    print_plugins_params &&
+    # Install common prerequisite
+    install_common_prereqs &&
+    # Install windows dependencies
+    install_windows_dependencies &&
+    # create_window_wagon
+    create_windows_wagon_package
+}
+
+function override_constraints_file() {
+    if [ ! -z "$CONSTRAINTS_FILE" ];then
+        cp -rf $CONSTRAINTS_FILE $PLUGIN_PATH/
+    fi
+}
+
+function clean_docker_resources() {
+  # This will clean all exited docker containers
```

Comment:
If it's stuck?
maybe use a specific name

Reply:
Sure will update that